pkg/ccextender/ccextender.py

```diff
@@ -1,13 +1,15 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python3.5
```

Comment:
Question: why not 3.7?

Reply:
Good point. Not quite sure I why decided on that either. Will change that!